src/visitor.cpp

```diff
@@ -41,6 +41,23 @@ namespace
 
         return cb(functor, container, {visitor_info::container_entity_exit, last_child});
     }
+
+    template <typename T>
+    bool handle_container(const cpp_entity& e, detail::visitor_callback_t cb, void* functor,
+                          detail::visitor_filter_t fcb, void* filter,
+                          bool last_child)
+    {
+        auto& container = static_cast<const T&>(e);
```

Comment:
Right, that was mistakenly removed. I'll try to come up with a unit test that should fail it it's not there.